classroom.rb

```diff
@@ -0,0 +1,22 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    student.classroom = self
+    @students << student
+  end
+end
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
+    @classroom = nil
+  end
```

Comment:
Great job @AyomipoMajek 🤝

- [ ] When setting the classroom for a student it also adds it to the classroom's students. Kindly add a `method that serves as the 'belongs-to' side (a student belongs to a classroom)`. Check out this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more guidance.

Reply:
Done, thank you